### DISCUSSION TITLE
use sd_dreambooth_extension to training lora, model file is too big

### DISCUSSION THREAD

**Discussion by segatecm** [d8ahazard/sd_dreambooth_extension#1132] [General]
I try to open the Lora option, and the training result file amost 8G.
But the kohya lora tool just 120M, Please tell me why?

**Comment by sailxjx** [comment]
Just save the lora, not the checkpoint, and reduce the unet and text encoder rank, 32/64 is enough.

**Reply by segatecm** [reply]
thanks, but how to it in sd_dreambooth_extension ?

**Comment by sailxjx** [comment]
In the save tab, <img width="552" alt="Screen Shot 2023-04-11 at 13 19 00" src="https://user-images.githubusercontent.com/909853/231063169-6f25fbab-517e-489b-b5f3-3fc4fbbe2d5c.png">
Uncheck `Generate a .ckpt file when saving during training.`
Check `Generate lora weights when saving during training.`

**Comment by segatecm** [comment]
Thanks!
